slice/Ice/EndpointTypes.ice

```diff
@@ -24,13 +24,31 @@
 module Ice
 {
 
-/// Determines the order in which the Ice run time uses the endpoints in a proxy when establishing a connection.
-local enum EndpointSelectionType
-{
-    /// <code>Random</code> causes the endpoints to be arranged in a random order.
-    Random,
-    /// <code>Ordered</code> forces the Ice run time to use the endpoints in the order they appeared in the proxy.
-    Ordered
-}
+/// Uniquely identifies TCP endpoints.
+const short TCPEndpointType = 1;
```

Comment:
The EndpointSelectionType enum is now in the EndpointSelectionType.ice file - a more appropriate name!